examples/fastrp-and-knn.ipynb

```diff
@@ -172,49 +261,126 @@
     "print(f\"Mean similarity: {result['similarityDistribution']['mean']}\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "f2bed240",
+   "metadata": {},
+   "source": [
+    "As we can see the mean similarity between nodes is quite high. This is due to the fact that we have a small example where there are no long paths between nodes leading to many similar FastRP node embeddings."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ff4175c9",
+   "metadata": {},
+   "source": [
+    "## Exploring the results\n",
+    "\n",
+    "Let us now inspect the results of our kNN call by using Cypher. We can use the `SIMILARITY` relationship type to filter out the relationships we are interested in. And since we just care about similarities between people for our product recommendation engine, we make sure to only match nodes with the `Person` label.\n",
+    "\n",
+    "Please see [the Cypher manual](https://neo4j.com/docs/cypher-manual/current/) for documentation on how to use Cypher."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "ebf3e425",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Rejoice in relationship in the Neo4j database!\n",
-    "query = \"\"\"\n",
+    "gds.run_cypher(\n",
+    "    \"\"\"\n",
     "        MATCH (p1:Person)-[r:SIMILAR]->(p2:Person)\n",
     "        RETURN p1.name AS person1, p2.name AS person2, r.score AS similarity\n",
     "        ORDER BY similarity DESCENDING, person1, person2\n",
-    "        \"\"\"\n",
+    "    \"\"\"\n",
+    ")"
+   ]
```

Comment:
more general as an idea, we could also think of using `mutate + streamRelationships` if they stay in the python world.  (lets not do it in this PR though)

Reply:
Nice idea, let's indeed consider it as a future improvement :)